packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -280,38 +280,38 @@ In the Okta event data object, the UserAgent object provides specifications abou
 
 | Property     | Description                                                                                                                                                                                                                                        | DataType       | Nullable |
 | ------------ | -------------------------------------------------------------------------------------------------                                                                                                                                                  | -------------- | -------- |
-| Browser      | If the client is a web browser, this field identifies the type of web browser (e.g. CHROME, FIREFOX)                                                                                                                                               | String         | TRUE     |
-| OS           | The [Operating System](https://en.wikipedia.org/wiki/Operating_system) the client runs on (e.g. Windows 10)                                                                                                                                        | String         | TRUE     |
-| RawUserAgent | A raw string representation of the user agent, formatted according to [section 5.5.3 of HTTP/1.1 Semantics and Content](https://tools.ietf.org/html/rfc7231#section-5.5.3). Both the `browser` and the `OS` fields can be derived from this field. | String         | TRUE     |
+| Browser      | If the client is a web browser, this field identifies the type of web browser (for example, CHROME, FIREFOX)                                                                                                                                               | String         | TRUE     |
+| OS           | The [operating system](https://en.wikipedia.org/wiki/Operating_system) that the client runs on (for example, Windows 10)                                                                                                                                        | String         | TRUE     |
+| RawUserAgent | A raw string representation of the user agent that is formatted according to [section 5.5.3 of HTTP/1.1 Semantics and Content](https://tools.ietf.org/html/rfc7231#section-5.5.3). Both the `browser` and the `OS` fields can be derived from this field. | String         | TRUE     |
 
 ### Request object
 
-The request object describes details related to the HTTP request that triggers this event, if available. When the event is not sourced to an http request, such as in the case of an automatic update on the Okta servers, the Request object will still exist, but the `ipChain` field will be empty.
+The Request object describes details that are related to the HTTP request that triggers this event, if available. When the event isn't sourced to an HTTP request, such as an automatic update on the Okta servers, the Request object still exists, but the `ipChain` field is empty.
```

Comment:
Same comment as previous regarding use of code style for objects, i.e. `request` object